udf_js/gunzip.sql

```diff
@@ -12,13 +12,24 @@ CREATE TEMP FUNCTION
   udf_js_gunzip (input BYTES)
   RETURNS STRING
   LANGUAGE js AS """
-    // converts a byte array to a string
+    /*  Input is either:
+     *    - A gzipped UTF-8 byte array
+     *    - A UTF-8 byte array
+     *
+     *  Outputs a string representation
+     *  of the byte array (gunzipped if
+     *  possible).
+     */
+
     function binary2String(byteArray) {
+        // converts a UTF-16 byte array to a string
```

Comment:
the comment before this function suggests that we should be decoded a UTF-**8** byte array to a string?
```suggestion
        // converts a UTF-8 byte array to a string
```

Reply:
[All javascript strings are internally UTF-16,](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCharCode) so this is unfortunately all a bit confusing to run

Reply:
I see where the confusion lies though, so I fixed the below documentation